### PR TITLE
Create an endpoint to refer to a specific form

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

File: forms/views.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from django.http import HttpRequest, HttpResponse, HttpResponseNotAllowed, JsonResponse
```
Comment (s373r) [inline forms/views.py:1]: squash your 2 commits into 1

File: forms/views.py [hunk 2]
```diff
@@ -11,3 +11,16 @@ def form_view(request: HttpRequest) -> HttpResponse:
     forms_json = list(forms_query_set.values())
 
     return JsonResponse(forms_json, safe=False)
+
+
+def show_form_id(request: HttpRequest, form_id: str) -> HttpResponse:
```
Comment (s373r) [inline forms/views.py:16]: rename `show_form_id()` -> `form_view_by_form_id()`

**Review by s373r — CHANGES_REQUESTED** [review]

File: forms/views.py [hunk 3]
```diff
@@ -11,3 +11,16 @@ def form_view(request: HttpRequest) -> HttpResponse:
...
+    if request.method != "GET":
+        return HttpResponseNotAllowed("GET")
+
+    forms_query_set = Form.objects.filter(form_id=form_id)
+
+    if bool(forms_query_set):
+        forms_json = list(forms_query_set.values())
+        return JsonResponse(forms_json, safe=False)
```
[the remark this answers is not in the document]
Comment (Serg-Bartosh) [reply forms/views.py:24]: [{'id': 1, 'name': 'test', 'description': '', 'questions': 'Do u like green', 'form_id': 'asdfghjkl;'}]

File: forms/views.py (hunk 1 above)
Comment (Serg-Bartosh) [reply forms/views.py:1]: Done

File: forms/views.py (hunk 2 above)
Comment (Serg-Bartosh) [reply forms/views.py]: DONE

File: forms/views.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Serg-Bartosh) [reply forms/views.py]: DONE

**Review by s373r — APPROVED** [review]
awesome!

**Pull request merged**